src/AudioManagerImpl.js

```diff
@@ -221,18 +222,25 @@ export default class AudioManagerImpl implements AudioManager {
         });
     }
 
-    playAnnouncement(messageIdSuffix: string, intl: IntlShape, messagePayload: any) {
-        if (this.announcementsEnabled) {
-            if (window.speechSynthesis.speaking || window.speechSynthesis.pending) {
-                window.speechSynthesis.cancel();
-            }
-            const messageId = "Announcement." + messageIdSuffix;
-            const toAnnounce = intl.formatMessage({ id: messageId}, messagePayload);
-            const utterance = new SpeechSynthesisUtterance(toAnnounce);
-            window.speechSynthesis.speak(utterance);
+    playFeedbackAnnouncement(message: string) {
+        if (this.audioFeedbackEnabled) {
+            this.cancelSpeech();
+            this.playStringMessage(message);
+        }
+    }
+
+    playPreviewAnnouncement(message: string) {
+        if (this.audioPreviewEnabled) {
+            this.cancelSpeech();
+            this.playStringMessage(message);
```

Comment:
I understand that the aria labels are being reused here.  I would suggest using an i18n template for preview announcements, something like `"{arialLabel} has focus"` or `{ariaLabel} highlighted`.